### PR TITLE
Feat/readme updates

### PR DESCRIPTION
**Pull request by krhoda**
This simplifies all READMEs and removes all out of date information. 

There are no code changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,51 +1,14 @@
...
-* [Demos!](https://github.com/spruceid/rebase/tree/main/demo) Which make use of all of the above.
+Several projects are hosted in the Rebase GitHub repo.
+* The [main library](./rust/rebase) written in Rust supports the creation of VCs from simpler structs, the signing of VCs using an `Issuer` abstraction, and witness flows for public issuers of VCs (along with their consuming clients).
+* The [witness library](./rust/rebase_witness_sdk) also written in Rust which supports specifically applications seeking to act as Issuer Witnesses or clients of Issuer Witnesses. Creates an opinionated implementation of the main library allowing users to quickly build on top of it.
+* The [client library](./js/rebase-client) published to NPM using Rust->WASM compilation, allowing in browser usage of the witness library in browser.
+* The [CloudFlare worker](./rust/rebase_cf_worker) which is open-source code base of the SpruceID Rebase Witness and serves as a working example / demo of the witness library.
```
Comment (obstropolos) [inline README.md:9]: `which is open-source code base` -> `which is an open-source codebase of...`

File: js/rebase-client/README.md [hunk 2]
```diff
@@ -1,137 +1,54 @@
...
-```
-$ npm i @rebase-xyz/rebase-client
-```
-
-## Common Usage:
-```javascript
-import { Client } from "@rebase-xyz/rebase-client";
-
-// const witnessUrl = ...
+Unfortunately, documentation for this project has not yet been written, but please see the [demo front-end project for a working example](../../demo/dapp/).
```
Comment (obstropolos) [inline js/rebase-client/README.md:5]: Documentation for the Rebase Client is coming soon. For now, please see the [demo front-end project for a working example](../../demo/dapp/).

File: rust/rebase/README.md [hunk 3]
```diff
@@ -1,121 +1,30 @@
 # Rebase
 
-Rebase offers an extensible, opinionated, trait-based approach to building credentialing systems on top of the foundation of [ssi](https://github.com/spruceid/ssi).
+Rebase is a library for building credentialing witnesses and clients to interact with them. Rebase is built on top of [SSI](https://github.com/spruceid/ssi) and uses [DID](https://www.w3.org/TR/did-core/)s and [Verifiable Credentials (VCs)](https://www.w3.org/TR/vc-data-model/) to create portable, specified credentials. Rebase provides abstractions for working with witnessed cryptographic signature-based attestations, linking cryptographic keys to traditional profiles and identities, and working with delegations to improve the experience for developers and users of applications with user-owned data models, along with other libraries available from [Spruce](), like [Kepler]() for storage.
```
Comment (obstropolos) [inline rust/rebase/README.md:3]: For [Spruce], let's link to our GH for now, [Kepler] we can unlink at the moment.

File: rust/rebase/README.md [hunk 4]
```diff
@@ -1,121 +1,30 @@
...
 
-The most comprehensive documentation of the foundational Rebase library and it's associated SDKs are found [at this site](https://www.spruceid.dev/rebase/rebase). A higher-level, more abstract overview is given here.
+Rebase has recently reached the stage that it is ready for publication. Unfortunately, the speed of development precludes comprehensive documentation. Hopefully this will be rectified shortly, but in the meantime here's a short overview.
```
Comment (obstropolos) [inline rust/rebase/README.md:5]: Remove / replace with -> "Here is a short overview of Rebase:

File: rust/rebase/README.md [hunk 5]
```diff
@@ -1,121 +1,30 @@
...
-## Other Resources
-
-[This is primary source of Rebase documentation](https://www.spruceid.dev/rebase/rebase), and should be able to answer questions unaddressed here.
-
-If you want to see examples in action, here's the [credential faucet](https://rebase.pages.dev/), that site's [codebase](https://github.com/spruceid/rebase/tree/main/demo/dapp), and the [codebase](https://github.com/spruceid/rebase/tree/main/demo/witness) of the Cloudflare worker witness service that backs that site.
-
-If you're looking for more information on how to run a witness service or how to create a client for an existing witness service, check out these two libraries ([witness SDK](https://github.com/spruceid/rebase/tree/main/rust/rebase_witness_sdk)), ([JS Client](https://github.com/spruceid/rebase/tree/main/js/rebase-client)). 
- 
\ No newline at end of file
+To see usage of this library, please look at the [Rebase Witness SDK]() and it's demo project, the [Rebase CloudFlare Worker]().
```
Comment (obstropolos) [inline rust/rebase/README.md:30]: Available links?

**Review by obstropolos — CHANGES_REQUESTED** [review]
A few copy changes

**Review by obstropolos — APPROVED** [review]

**Review by anukritidata — APPROVED** [review]

**Pull request merged**